tests/queries/0_stateless/02114_basic_acess_auth.sql

```diff
@@ -0,0 +1 @@
+select * FROM url('https://guest:guest@jigsaw.w3.org/HTTP/Basic/','RawBLOB', 'a String');
```

Comment:
Also I believe this test will not work in ci from docker.

Reply:
Even if it will be ok, this is not good.
Can you please try to use http server in python in stateless tests like here https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/0_stateless/00646_url_engine.python?

Reply:
OK, I will write a python script to run this test later.